src/Uno.UWP/Devices/Radios/Radio.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.Devices.Radios
+{
+	public partial class Radio

```

Comment:
The Radio class is not implemented, so there's no point removing the not implemented attributes.

Reply:
This structs are common for all platforms, I extracted it to this PR.
I have ready implementation of GetRadiosAsync(), for Android only, and I want it to be in another PR.

Reply:
That's fine, then you can add this class to this PR.

Reply:
Android GetRadiosAsync() is in #5596.
Why mix all platforms PRs with single platforms PRs?

All my single platform PRs are still unmerged, for over year now... But "all platforms" structs from these PRs, extracted to simpler PRs, are being merged. Maybe as help for others, who would write same PRs as I, but multiplatform.

Reply:
Please remove this class from this PR, and add it to the other PR. This class is not implemented until the other PR is merged.

Reply:
As #5596 included all changes from this PR (I didn't intend to, but git/vstudio make it so) , maybe skip this PR and deal only with #5596?

Reply:
All properties of class Radio are implemented here (#5589), code is not implemented as code is platform related (and properties are same for all platforms).
But, when I restrict this PR to enums only, would it be accepted/merged?